### ISSUE TITLE
logging to multiple files breaks when using :logsize-rotate

### ISSUE THREAD

**Issue by faxm0dem** [riemann/riemann#551]
It seems that when trying to set up file-size based rotation when logging to multiple files, no logs are written at all. No error message on the console neither.

```Clojure
(logging/init {:files [{:path "/var/log/riemann/riemann.log" :layout :riemann}] :logsize-rotate 1000000})
```

**Comment by juise** [comment]
Any changes?

Looks like it's bug in https://github.com/aphyr/riemann/blob/master/src/riemann/logging.clj#L124.

**Comment by pyr** [comment]
fixed in #598
